README.md

```diff
@@ -2,208 +2,394 @@
+  const favoriteFruit = "yuzu";
+  console.log("A capybara's favorite bath fruit is: ", favoriteFruit); // => A capybara's favorite bath fruit is: yuzu
+} else {
+  console.log(favoriteFruit); // => ReferenceError: favoriteFruit is not defined
+  return "No animal here, sorry!";
+}
 
-myOtherVar;
-// Uncaught ReferenceError: myOtherVar is not defined
+console.log(favoriteFruit); // => ReferenceError: favoriteFruit is not defined
 ```
 
-This is yet another reason to **never use `var`**. As long as you stick to
-declaring variables with `const` and `let`, what happens in block stays in
-block.
+Even though the `if/else` blocks are part of the same conditional, each
+statement has its own block. Both the `if` and `else` statement have their own
+sets of curly braces `{}`, and therefore have separate scopes. Anything defined
+within the `if` statement, such as `favoriteFruit`, does not exist outside of
+that block.
+
+## Gotcha's to Be Aware Of
+
+Now that we've covered the main three scopes, there are two gotcha's you should
+be aware of moving forward. Both have to do with how you declare a variable.
 
-## The global gotcha
+All of our examples thus far have either used `const` or `let` to declare a
+variable. For best practices, we recommend you always use one of the two.
+However, you may encounter legacy code that uses `var` or doesn't use a keyword
+at all. In such cases, those variables do not follow the scope rules we just
+learned about.
 
-In a perfect world, you'd always remember to declare new variables with `const`
-and `let`, and you'd never run into any weird scoping issues. However, it's
-inevitable that at some point you're going to forget the `const` or `let` and
-accidentally do something like:
+## The `var` Gotcha
+
+If you declare a variable with **`var`**, that variable **does not get block
+scoped**. Let's demonstrate this with a modified version of our `if` statement
+example from before:
 
 ```js
-firstName = "Ada";
+const animal = "capybara";
+
+if (animal === "capybara") {
+  var favoriteFruit = "yuzu";
+  console.log("A capybara's favorite bath fruit is: ", favoriteFruit); // => A capybara's favorite bath fruit is: yuzu
+}
+
+console.log(favoriteFruit); // => yuzu
 ```
 
-Variables created without a `const`, `let`, or `var` keyword are **always
-globally-scoped**, regardless of where they sit in your code. If you create one
-inside of a block, it's still available globally:
+Because it was defined with `var`, we are able to access `favoriteFruit` outside
+of the `if` statement it was defined in.
+
+However, `var` does **get function scoped**. For example, let's wrap our `if`
+statement inside a function:
 
 ```js
-if (true) {
-  lastName = "Lovelace";
+const animal = "capybara";
+
+function logAnimalFruit() {
+  if (animal === "capybara") {
+    var favoriteFruit = "yuzu";
+    console.log("A capybara's favorite bath fruit is: ", favoriteFruit); // => A capybara's favorite bath fruit is: yuzu
```

Comment:
Maybe take out the console.log for this example? It may get in the way of the main point here